### ISSUE TITLE
Incompatible with Gauges and Switches

### ISSUE THREAD

**Issue by X-XLebyshek** [SteveKunG/FishOfThieves#43]
It crashes with Gauges and Switches. Without it Minecraft loads normally.
**Version:** 1.19.2
**Crash Report:** https://gist.github.com/X-XLebyshek/1b58889f8b0e91835f55db1dabdf9008
Fish of Thieves: 2.2.4
Gauges and Switches: 1.2.19
Forge: 43.3.5
When I first loaded Minecraft with both mods installed, it also worked. The issue appeared after the second launch, I don't know why.

**Comment by SteveKunG** [comment]
Can you upload full log? File located at .minecraft/logs/latest.log.

**Comment by X-XLebyshek** [comment]
Here you are
https://gist.github.com/X-XLebyshek/2a63f1405c6f11188508740e0f628ece

**Comment by SteveKunG** [comment]
One of these mods may cause the issues. Gauges and Switches and I Like Wood. But I downloaded mods and tested on my machine seems working properly.

```
[21:35:41] [modloading-worker-0/ERROR]: Exception caught during firing event: Registry Object not present: rsgauges:industrial_white_blinking_led
	Index: 1
	Listeners:
		0: NORMAL
		1: ASM: class wile.rsgauges.ModRsGauges$ForgeEvents onConfigLoad(Lnet/minecraftforge/fml/event/config/ModConfigEvent$Loading;)V
java.lang.NullPointerException: Registry Object not present: rsgauges:industrial_white_blinking_led
```

```
[21:35:41] [modloading-worker-0/ERROR]: Failed to create mod instance. ModID: ilikewood, class yamahari.ilikewood.ILikeWood
java.lang.NullPointerException: Registry Object not present: rsgauges:industrial_white_blinking_led
```

**Comment by X-XLebyshek** [comment]
I like wood works fine, its Gauges and Switches. How do I fix this? Or does that mean that I should just choose between these mods?

**Comment by X-XLebyshek** [comment]
I just launched Minecraft without fish of thieves, but with gauges and switches and it crashed with i like wood. That means that fish of thieves didn't cause the crash, I think

**Comment by SteveKunG** [comment]
Fixed in 2.2.5 by https://github.com/SteveKunG/FishOfThieves/commit/573a9c39e9060a0f6d6a0275c8fcfa345b61df1e. The mod shouldn't hard crash anymore if it can't find mod dependencies.
